src/SearchAggregator/Controllers/PackagesController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNet.Mvc;
```

Comment:
Don't want to merge unrelated code into dev. You can keep working in your branch